src/pages/portfolio/AccountOverviewSection.tsx

```diff
@@ -0,0 +1,182 @@
+import { useCallback } from 'react';
+
+import { shallowEqual } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+
+import { ButtonAction } from '@/constants/buttons';
+import { STRING_KEYS } from '@/constants/localization';
+import { AppRoute } from '@/constants/routes';
+import { ColorToken } from '@/constants/styles/base';
+
+import { useBreakpoints } from '@/hooks/useBreakpoints';
+import { useStringGetter } from '@/hooks/useStringGetter';
+import { useLoadedVaultAccount, useLoadedVaultDetails } from '@/hooks/vaultsHooks';
+
+import { Button } from '@/components/Button';
+import { Output, OutputType } from '@/components/Output';
+import { NewTag, Tag, TagSign, TagType } from '@/components/Tag';
+import { WithLabel } from '@/components/WithLabel';
+
+import { getSubaccount } from '@/state/accountSelectors';
+import { useAppSelector } from '@/state/appTypes';
+
+import { runIf } from '@/lib/do';
+import { isTruthy } from '@/lib/isTruthy';
+import { testFlags } from '@/lib/testFlags';
+import { orEmptyObj } from '@/lib/typeUtils';
+
+export const MegavaultYieldTag = () => {
+  const stringGetter = useStringGetter();
+  const vault = useLoadedVaultDetails().data;
+  if (vault?.thirtyDayReturnPercent == null || vault.thirtyDayReturnPercent < 0.02) {
```

Comment:
can we extract 0.02 as a constant somewhere (constants/vaults)?

Reply:
pulled it out into a constant in this file since it's really only relevant here.